docs/executor.rst

```diff
@@ -85,6 +85,30 @@ Resource requests and other job characteristics can be controlled via the follow
 * :ref:`process-time`
 
 
+.. _flux-executor:
+
+Flux Framework Executor
+=======================
+
+The ``flux`` executor allows you to run your pipeline script by using `Flux Framework <https://flux-framework.org>`_.
+
+Akin to slurm, Nextflow manages each process as a separate job that is submitted to the cluster by using the ``flux submit`` command.
+
+Being so, the pipeline must be launched from a node where the ``sbatch`` command is available, that is, in a common usage
+scenario, the cluster `head` node.
```

Comment:
`s/sbatch/flux/`. However, this line can probably be removed because the `flux(1)` command should be available everywhere on a cluster or job running flux.